### PR TITLE
Add parenthesis after bare traits

### PR DESCRIPTION
**Pull request by corranwebster**
Simple search and replace for `Int` to `Int()` and similar.

Edit: And it looks like VS Code has _helpfully_ removed some excess whitespace as well.

### CODE REVIEW

File: docs/source/traitsui_user_manual/factories_advanced_extra.rst [hunk 1]
```diff
@@ -172,7 +172,7 @@ with associated key bindings, and a button that invokes the key binding editor.
     class KBCodeExample ( HasPrivateTraits ):
 
         code   = Code
-        status = Str
+        status = Str()
```
Comment (mdickinson) [inline docs/source/traitsui_user_manual/factories_advanced_extra.rst]: Another search-and-replace for `Code` needed?

Comment (corranwebster) [reply docs/source/traitsui_user_manual/factories_advanced_extra.rst]: Yep - searched for only _one_ space.  *sigh*

**Comment by corranwebster** [comment]
Fixed a few more.  Mypy should complain about missing parenthesis with the traits stubs installed (although in a not entirely intuitive way).

**Comment by corranwebster** [comment]
Travis CI tests are in fact passing, just not showing here.  Going to merge.

**Pull request merged**